docs/user/troubleshooting/03-46-init-containers-cant-access-network.md

```diff
@@ -5,16 +5,13 @@
 Init containers can't access the network.
 
 ## Cause
+If Istio injection is enabled, the `istio-proxy` container intercepts all network traffic from all containers. By default, the Istio module injects `istio-proxy` containers as native sidecars. However, you can also set the annotation `sidecar.istio.io/nativeSidecar` annotation to `"false"` for a specific Pod. This annotation overwrites the default setting and indicates that instead of native sidecars, the annotated Pod must be injected with a regular sidecar container.
```

Comment:
```suggestion
If Istio injection is enabled, the `istio-proxy` container intercepts all network traffic from all containers. By default, the Istio module injects `istio-proxy` containers as native sidecars. However, you can also set the annotation `sidecar.istio.io/nativeSidecar` to `"false"` for a specific Pod. This annotation overwrites the default setting and indicates that instead of native sidecars, the annotated Pod must be injected with a regular sidecar container.
```